# This srv file defines the service
# for resetting the robot to home configuration (armType specified) 
# after finish a task

string armType
bool isLabeledRoadmapUsed
---
bool success
uniform_object_rearrangement/ArmTrajectory resetHome_trajectory